.github/workflows/plugin-check.yml

```diff
@@ -0,0 +1,53 @@
+name: 'Plugin Check'
+on:
+  pull_request:
+  push:
+    branches:
+    - main
+    - trunk
+    - 'release/**'
+
+
+jobs:
+  prepare:
+    name: "Build and check plugin"
+    # Don't run on forks.
+    if: github.repository_owner == 'Yoast'
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.2
+          coverage: none
+
+      - name: Set up node and enable caching of dependencies
+        uses: actions/setup-node@v3
+        with:
+          node-version: '14'
+          cache: 'yarn'
+
+      - name: Install Composer dependencies and run the prefixing script
+        uses: ramsey/composer-install@v2
+        with:
+          # Bust the cache at least once a week - output format: YYYY-MM-DD.
+          custom-cache-suffix: $(/bin/date -u --date='last Mon' "+%F")
+
+      - name: Yarn install
+        run: yarn install
+
+      - name: "Grunt: create artifact"
+        run: grunt artifact
+
+      - name: "Copy composer.json"
+        run: cp composer.json ./artifact
```

Comment:
Probably not needed

Reply:
removed in 48733cd354c2ef7ba771dff7601b9fb5598af912